src/Slider/Slider.tsx

```diff
@@ -50,29 +56,50 @@ const Slider: React.FunctionComponent<SliderProps> = (props: SliderProps): React
             });
             setLabelsPositions(positions);
         }
-    }, [props.labels]);
+    }, [props.labels, min, max]);
+
+    React.useEffect(() => setThumbPosition(getPercentage()), [props.value, min, max, size]);
+
+    /**
+     * Finds the size between two numbers
+     * @param {number} minValue The minimum value
+     * @param {number} maxValue The maximum value
+     * @returns {number} The size
+     */
+    function getSize(minValue: number, maxValue: number): number {
+        if (maxValue > minValue) {
+            return maxValue - minValue;
+        } else { // Will calculate the size anyway, but it will show a warning since the min is larger than the max
+            console.warn(`The max value of the slider should be larger than the min value (Max:${max}, Min: ${min}`);
+            return minValue - maxValue;
+        }
+    }
```

Comment:
```
        if (maxValue <= minValue) {
            console.warn(`The max value of the slider should be larger than the min value (Max:${max}, Min: ${min}`);
        } 
        return minValue - maxValue;
```

Reply:
Your suggested changes won't work since I'm returning a value in both `if` and `else` where each one is different